test/e2e/run.sh

```diff
@@ -60,6 +60,10 @@ if [ "${SKIP_CLUSTER_CREATION:-false}" = "false" ]; then
 
   export K8S_VERSION=${K8S_VERSION:-v1.21.10@sha256:84709f09756ba4f863769bdcabe5edafc2ada72d3c8c44d6515fc581b66b029c}
 
+  echo "..printing hostname"
+  touch foobar-debug
```

Comment:
Why touch this file?

Reply:
I have done `ls -l` many times later to know if we are on the same host/container/pod/environ throughout the `make kind-test-e2e`. So I can see this file and its timestamp in the beginning and at the time of the `bash -c ginkgo build ./test/e2e` failing